client/payment-details/dispute-details/dispute-awaiting-response-details.tsx

```diff
@@ -337,6 +339,8 @@ const DisputeAwaitingResponseDetails: React.FC< Props > = ( {
 													{
 														dispute_status:
 															dispute.status,
+														on_page:
+															'transaction_details',
 													}
```

Comment:
@brucealdridge Can I get your opinion on whether this event property is necessary for the "view modal" button event?

It seems redundant since this is only every rendered on the transaction details screen.

Reply:
I've created an issue for this #7474.